### ISSUE TITLE
loading a sketch.js file?

### ISSUE THREAD

**Issue by luisaph** [toolness/p5.js-widget#59]
I would like to load code from an external file, like this: 

```html
<script type="text/p5" src="sketch.js"  ></script>
```

Curious: is this feature supported? 

When I try the above, instead of my sketch I see what seems to be a default one:
<img width="605" alt="screen shot 2016-06-30 at 10 45 14 am" src="https://cloud.githubusercontent.com/assets/295879/16492448/d8eb8706-3eaf-11e6-9ec7-00c5c2856ea6.png">

Thanks!

**Comment by toolness** [comment]
Hey, there isn't currently a way to do this, but there totally should be!

How soon do you need the feature?

**Comment by luisaph** [comment]
I would use for a project that's due in two weeks, but it's not a critical thing :)

**Comment by toolness** [comment]
Ok, I've implemented basic support for it! It's documented here:

https://toolness.github.io/p5.js-widget/#src

So your example snippet should work now.

I also tried to handle error conditions gracefully, so if the file couldn't be retrieved, you'll see something like this:

![404](https://cloud.githubusercontent.com/assets/124687/16545228/560c119e-40f2-11e6-93ef-6e3f83614555.png)

Another weird situation I've seen beginners run into happens when they've accidentally specified *both* a `src` attribute *and* an inline script, e.g.:

```html
<script type="text/p5" src="static/my-example.js">
function setup() {
  // Do stuff here...
}
</script>
```

With normal script tags, browsers will use either the `src` attribute or the inline script (I forget which), which often leaves the user confused when they edit the *other* one and don't see any changes. So instead, p5.js-widget treats this kind of situation as an error and displays this:

![src-and-inline-error](https://cloud.githubusercontent.com/assets/124687/16545235/9a51847e-40f2-11e6-90b2-72e80bf967d0.png)

Anyhow, let me know if this works okay for you! Thanks for filing the issue.
